app/controllers/openid_application_controller.rb

```diff
@@ -109,6 +123,14 @@ def expected_key(token)
     decoded_token = JWT.decode(token, nil, false, algorithm: 'RS256')
     kid = decoded_token[1]['kid']
     OIDC::KeyService.get_key(kid)
+  rescue JWT::ExpiredSignature
+    raise error_klass('Validation error: token has expired')
+  rescue JWT::DecodeError
+    raise error_klass('Validation error: token could not be validated')
```

Comment:
```suggestion
    raise error_klass('Validation error: token could not be decoded')
```
Bit of a cyclical error message as written :thinking: , proposed a new change